src/cpp/src/llm_pipeline.cpp

```diff
@@ -36,20 +36,18 @@ ov::genai::EncodedResults multinominal_decoding(
 );
 
 EncodedResults beam_search(
-    ov::InferRequest& lm, 
```

Comment:
It's been done automatically again, can return whitespaces back if there is a sense in doing this